neonvm/runner/main.go

```diff
@@ -574,6 +578,14 @@ func main() {
 	// disk details
 	qemuCmd = append(qemuCmd, "-drive", fmt.Sprintf("id=rootdisk,file=%s,if=virtio,media=disk,index=0,cache=none", rootDiskPath))
 	qemuCmd = append(qemuCmd, "-drive", fmt.Sprintf("id=runtime,file=%s,if=virtio,media=cdrom,readonly=on,cache=none", runtimeDiskPath))
+	{
+		diskName := "ssh-authorized-keys"
+		diskPath := fmt.Sprintf("%s/%s.qcow2", mountedDiskPath, "ssh-authorized-keys")
```

Comment:
1. ISO9660 is not `.qcow2`, probably `.iso`
2. "ssh-authorized-keys" ->  diskName. And it probably can be a global constant.

Reply:
resolved in 8bcd4de3773cd3089993246647a915bc1d7af8dd and 63d86fa16fed9b79ccd43de7ae242a2a504e290f.